fixit/common/generate_pyre_fixtures.py

```diff
@@ -59,6 +59,7 @@ def gen_types_for_test_case(source_code: str, dest_path: Path) -> None:
         if "error" in data:
             raise PyreQueryError(cmd, data["error"])
         data = data["response"][0]
+        # pyre-fixme[35]: Target cannot be annotated.
```

Comment:
This should be easy to fix too, just rename occurrences of `data` before this line to `raw_data`